### PR TITLE
Overlay with explaination added

### PR DESCRIPTION
**Pull request by siddhikhapare**

### CODE REVIEW

File: pkg/dashboard/static/index.html [hunk 1]
```diff
@@ -74,7 +74,27 @@
                         style="height: 1.2rem; vertical-align: text-bottom; filter: grayscale(00%);"></a>
             </div>
             <div class="separator-vertical"><span></span></div>
-            <i class="btn bi-power text-muted p-2 m-1 mx-2" title="Shut down the Helm Dashboard application"></i>
+            <!-- <i class="btn bi-power text-muted p-2 m-1 mx-2" title="Shut down the Helm Dashboard application"></i> -->
+            <button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#Modal">
```
Comment (undera) [inline pkg/dashboard/static/index.html:78]: Let's keep the original button, the purpose of the issue were only to add the modal. Attaching the handler to `i.bi-power` would work just fine

**Review by undera — CHANGES_REQUESTED** [review]
Please address my comments

**Comment by hdm23061993** [comment]
Fixes #105

**Comment by siddhikhapare** [comment]
Sure @undera

**Pull request closed without merging**